.github/workflows/linux-lint.yaml

```diff
@@ -0,0 +1,40 @@
+name: Linux Lint
```

Comment:
Let's merge it with `linux-ci.yml` and only run on master (it takes more than a hour)

Reply:
Linux-ci is sill needed on PRs for feedback -- checking build and coverage.  Linting can be separate, but run only on master.

Reply:
Combined into one workflow file